subsys/net/ip/tcp.c

```diff
@@ -1857,6 +1859,40 @@ static void tcp_fin_timeout(struct k_work *work)
 	(void)tcp_conn_close(conn, -ETIMEDOUT);
 }
 
+static void tcp_last_ack_timeout(struct k_work *work)
```

Comment:
Maybe make these `inline`?

Reply:
Most probably the compiler is able to figure out itself what functions to inline or not. There is an old discussion about this in Linux kernel https://www.kernel.org/doc/local/inline.html and relevant links in that doc to LWN.